### ISSUE TITLE
[Request] (Treadspear) These 6 or more options can make chat more compact

### ISSUE THREAD

**Issue by ZemyCode** [SevenTV/Extension#480]
### Discussed in https://github.com/SevenTV/Extension/discussions/467

<div type='discussions-op-text'>

<sup>Originally posted by **Treadspear** April  5, 2023</sup>
Please will you consider add these option for the chat:

- add/remove hype train
- add/remove top gifted
- add/remove pinned
   - option to show pinned for 30secons
- add/remove "STREAM CHAT" title and "COMMUNITY" button
- add/remove chat button, channel points button, twitch chat settings icon.

Here is a before and after photo of what chat looks like with all these options disabled.

Before:  https://1drv.ms/i/s!Avv6335OlHOmgTuM4nkO564xvIpw?e=OC5xcA [Microsoft OneDrive]
![Untitled](https://user-images.githubusercontent.com/126298418/230256325-55bb8e9a-df4e-49a0-b961-150bf0d7998f.png)

After: https://1drv.ms/i/s!Avv6335OlHOmgTw-eLOU4-EwaL6b?e=1EJR1Z [Microsoft OneDrive]
![Untit4led](https://user-images.githubusercontent.com/126298418/230256321-a3b4cac8-f0ed-4ed8-a8e0-80ea273c9fe2.png)

This would be great for those who have a single monitor but multitask with another window.
Let me know how this may be useful for you.

Thank you</div>

**Comment by ZemyCode** [comment]
Hi, I am working on this feature so I decided to make an issue for it since it was posted in discussion.

**Comment by ZemyCode** [comment]
This issue is mostly addressed in PR https://github.com/SevenTV/Extension/pull/481. The desired UI Treadspear had screenshotted in his description is achievable with the newly added hidden elements feature. However, it was not implemented in a way where specifically hype train or specifically pinned message can be hidden, it will just hide any community highlights in general. There could be future work addressing this, and allowing for hiding specifically just hype train, etc.
